src/main/java/mods/eln/Eln.java

```diff
@@ -5241,6 +5251,16 @@ void recipeTurbine() {
 				Character.valueOf('A'), "blockAluminum",
 				Character.valueOf('M'), findItemStack("Advanced Machine Block")
 		);
+		addRecipe(findItemStack("Gas Turbine"),
+				"msH",
+				"sSs",
+				" M ",
+				Character.valueOf('m'), findItemStack("Advanced Electrical Motor"),
+				Character.valueOf('H'), findItemStack("Copper Thermal Cable"),
+				Character.valueOf('s'), "ingotSteel",
```

Comment:
afaik ingotSteel does not exists in Vanilla Minecraft + EA. Imho recipes should only contain Minecraft stock items and EA items.